src/sst/elements/prospero/proscpu.h

```diff
@@ -28,9 +28,11 @@
 #include "prosreader.h"
 #include "prosmemmgr.h"
 
+#ifndef HAVE_PIN3
```

Comment:
@allevin This file is not part of the pintool so it can use zlib. It shouldn't have any guards about PIN3.

Reply:
Fixed this one